### PR TITLE
Adjust SplittedLayout with button size and colors #89

### LINKED ISSUE #89: Adjust SplittedLayout with button size and colors
**Adjust button and colors in home, check and ´confirm´**
Change the button size and colors.
It looks like this in desktop:
Captura de Pantalla 2021-10-12 a la(s) 12 23 21 p  m

It looks like this in mobile:
Captura de Pantalla 2021-10-12 a la(s) 12 24 15 p  m

It should look like this:
Desktop:
Captura de Pantalla 2021-10-12 a la(s) 12 30 55 p  m

Mobile:
Captura de Pantalla 2021-10-12 a la(s) 12 30 29 p  m

### PR DESCRIPTION
**Pull request by ginnaaceror**
Adjust SplittedLayout with button size and colors.

### CODE REVIEW

File: src/components/SplittedLayout.jsx [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+    position: fixed;
+    margin-left: -50px;
+  }
+`
+const MainSection = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  height: 100%;
```
Comment (Israel-Laguan) [inline src/components/SplittedLayout.jsx]: please add max-width: 1220px; or similar

It looks too weird on wide screens
image

File: src/components/SplittedLayout.jsx [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+  @media (min-width: 960px) {
+    flex-direction: row;
+  }
+`
+
+const Header = styled.div`
+  display: flex;
+  flex-direction: column;
+  background: var(--primary);
+  border-radius: 0 0 25px 25px;
```
Comment (Israel-Laguan) [inline src/components/SplittedLayout.jsx]: please add max-width: 1220px; or similar

It looks too weird on wide screens
image

File: src/components/SplittedLayout.jsx [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+  }
+`
+const MainSection = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  height: 100%;
+  @media (min-width: 960px) {
+    background: var(--secondary);
```
Comment (Israel-Laguan) [inline src/components/SplittedLayout.jsx]: Can you make something to fix that Form title is not visible?
image

**Comment by Israel-Laguan** [comment]
https://user-images.githubusercontent.com/36519478/137034390-7c869212-ee2f-4053-8b53-ef091a7e6c26.mp4

Please check this video how media query are not behaving as expected

**Comment by AlejandroSalcido** [comment]
>  ginna.mp4 
> Please check this video how media query are not behaving as expected

im working on it!

File: src/pages/Check.jsx [hunk 4]
```diff
@@ -29,7 +29,7 @@ const Check = () => {
       </Header>
       <MainSection>
         <div className="check-ticket">
-          <Text color="secondary" bold uppercase>
+          <Text color="secondary" bold uppercase colorDesk>
```
Comment (Israel-Laguan) [inline src/pages/Check.jsx]: Please note colorDesk is the same as say color="primary" so this new prop is not really needed

Comment (Israel-Laguan) [reply src/pages/Check.jsx]: I changed it so no worry about it

**Review by Israel-Laguan — APPROVED** [review]

**Pull request merged**